src/main/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/indexstatemanagement/ManagedIndexRunner.kt

```diff
@@ -165,6 +166,8 @@ object ManagedIndexRunner : ScheduledJobRunner,
         clusterService.clusterSettings.addSettingsUpdateConsumer(ALLOW_LIST) {
             allowList = it
         }
+
+        hostDenyList = settings.getAsList(ManagedIndexSettings.HOST_DENY_LIST)
```

Comment:
the reason not add updateconsumer for this setting is because this can only be set in elasticsearch.yml? cannot change dynamiclly?

Reply:
Yeah its set as final in alerting plugin